climate-monitoring-core/src/main/java/climatemonitoring/core/View.java

```diff
@@ -1,6 +1,46 @@
+/*
+
+Alessandro della Frattina 753073 VA
+Cristian Capiferri 752918 VA
+Francesco Lops 753175 VA
+Dariia Sniezhko 753057 VA
+
+*/
+
 package climatemonitoring.core;
 
-public class View {
+/**
+ * The View class manages the state of a view and handles rendering operations based on the configuration (headless or GUI)
+ * @author francescolops
```

Comment:
This is ok, just put a new line between the class description and the `@author` tag